src/router.js

```diff
@@ -14,7 +14,7 @@ router.post('/register', (req, res, next) => {
 
 // login
 
-router.get('/login', (req, res, next) => {
+router.post('/login', (req, res, next) => {
   res.sendStatus(200);
   next();
 });
```

Comment:
For line 50 regarding router update, same issue here. preferable to label the route by the product name so that it can easily be referenced ie. router.put('/clothes')

Reply:
I agree with you and i will have it changed.  Thank you for your suggestion.